lib/src/link/types.ts

```diff
@@ -52,19 +48,19 @@ export type LinkTextProps = LinkCommonProps & {
    */
   text: string;
   /**
-   * Element used as the icon that will be placed next to the link text.
+   * Element or path used as the icon that will be placed next to the link text.
    */
-  icon?: SVG;
+  icon?: string | SVG;
```

Comment:
Remove from `type SVG` the function type: `| React.FunctionComponent<React.SVGProps<SVGSVGElement>>`.

Reply:
Done ✌🏼